apps/docs/src/app/cli/commands/page.mdx

```diff
@@ -241,3 +241,24 @@ Immediately kill a running sandbox.
 e2b sandbox kill <sandboxID>
 ```
 
+## `sandbox logs`
+Start printing logs from the specified sandbox.
+If the sandbox is running new logs will be streamed to the terminal.
```

Comment:
Do you think streaming by default is the best? I would add a flag to follow as everybody does.